### ISSUE TITLE
`state_capital:` true?

### ISSUE THREAD

**Issue by patriciogonzalezvivo** [tangrams/tron-style#124]
Hi... isn't [here](https://github.com/tangrams/tron-style/blob/gh-pages/layers/places.yaml#L182) in `state_capital` be `true/false` and not `yes` ??

**Comment by nvkelso** [comment]
It's region_capital now, with true.

**Comment by patriciogonzalezvivo** [comment]
ok... I will change all the `yes` for `true` in roads.yaml that's ok?

**Comment by nvkelso** [comment]
Probably.

**Comment by nvkelso** [comment]
@patriciogonzalezvivo no, roads.yaml only has `oneway: yes`, which is still the expected values (they are a little complicated, not just booleans).

But in places.yaml `country_capital` should be `true` as well as the `region_capital` (rename it from `state_capital`).

**Comment by patriciogonzalezvivo** [comment]
Done https://github.com/tangrams/tron-style/commit/079abfca6c300316e8d8581d3ad3b24eac344c4e
